burglebros.js

```diff
@@ -1154,6 +1169,72 @@ function (dojo, declare) {
             });
         },
 
+        setupCrystalBallCards: function(event_cards) {
+            this.elementDragged = null;
+            for (var i in event_cards) {
+                var card = event_cards[i];
+                dojo.place( this.eventCardHtml(card, card.id, 'dnd_card', true), 'crystal_ball_cards' );
+                this.addCardTooltip(card, 'event_card_dialog_' + card.id);
+            }
+            if (this.isCurrentPlayerActive()) {
```

Comment:
You probably want to try this out on a mobile device. Generally drag and drop is not the best experience there.

Reply:
ok, i'll implement vertical arrows with select / clicks to implement card move